desktop/static/main.ts

```diff
@@ -279,6 +280,12 @@ function createWindow() {
     minWidth: 800,
     minHeight: 600,
     center: true,
+    // The app icon is defined in package.json by default.
+    // When building Linux zip, it must be defined here or else it won't work.
+    icon:
+      os.platform() === 'linux'
+        ? path.join(__dirname, 'icons/app_64x64.png')
```

Comment:
Have you checked if this also works for both development and release builds? There are times where you need to resolve the static path differently because of the ASAR compression.

Reply:
I'm not really sure how to know if I'm running a development or release build. However, the icon works in both ways:
* `yarn start`
* `yarn build --linux --version 0.82.1`

Reply:
Thanks! Just wanted to check that you tested both cases. :)